org.eclipse.scout.rt.ui.html/src/main/java/org/eclipse/scout/rt/ui/html/json/UnloadRequestHandler.java

```diff
@@ -57,8 +57,14 @@ public boolean handlePost(HttpServletRequest req, HttpServletResponse resp) thro
   protected void handleUnloadRequest(HttpServletRequest req, HttpServletResponse resp, String uiSessionId) {
     LOG.info("Unloading UI session with ID {} (requested by UI)", uiSessionId);
 
-    final HttpSession httpSession = req.getSession();
-    final ISessionStore sessionStore = BEANS.get(HttpSessionHelper.class).getSessionStore(httpSession);
+    final HttpSession httpSession = req.getSession(false);
```

Comment:
remove the UNLOAD_PATH_PATTERN regex in this class and use the one from the TrivialAccessController. Furthermore, I would remove all these unnecessary "final" keywords in this class to remove noise.

Reply:
I agree, it should be the other way round. But this is not possible. I prefer having it in one place. Otherwise they will diverge. 
If anybody would like to reimplement the authentitcation, this still works if the constant is only there.
As an alternative, all the constants (also the others like "login") could be extracted to an extra interface. Then it can be used by both clients.